src/main/webapp/user-page.html

```diff
@@ -36,9 +39,17 @@ <h1 id="page-title">User Page</h1>
       Enter a new message:
       <br/>
       <textarea name="text" id="message-input"></textarea>
+     
```

Comment:
In most cases, don't put multiple blank lines unless required by the style guide.